packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -393,6 +450,83 @@ describe('HttpHandler', () => {
     }
   });
 
+  context('multipart/form-data', () => {
+    beforeEach(givenBodyParamController);
+
+    it('supports file uploads', async () => {
+      const FIXTURES = path.resolve(__dirname, '../../../fixtures');
+      const res = await client
+        .post('/show-body')
+        .field('user', 'john')
+        .field('email', 'john@example.com')
+        .attach('certFile', path.resolve(FIXTURES, 'cert.pem'), {
+          filename: 'cert.pem',
+          contentType: 'multipart/form-data',
+        })
+        .expect(200);
+      expect(res.body.files[0]).containEql({
+        fieldname: 'certFile',
+        originalname: 'cert.pem',
+        mimetype: 'multipart/form-data',
+      });
+    });
+
+    function givenBodyParamController() {
+      const spec = anOpenApiSpec()
+        .withOperation('post', '/show-body', {
+          'x-operation-name': 'showBody',
+          requestBody: <RequestBodyObject>{
+            description: 'multipart/form-data value.',
+            required: true,
+            content: {
+              'multipart/form-data': {
+                // Skip body parsing
+                'x-skip-body-parsing': true,
```

Comment:
Please add a simpler test to verify `x-skip-body-parsing` feature. One that has less moving parts and will be easier to debug if anything goes wrong.